src/components/CreateProfile/LocationPreferences/LocationPreferences.tsx

```diff
@@ -41,37 +22,51 @@ const LocationPreferences = () => {
       </div>
       <div className={styles.right}>
         <div>
-          <TextInput
-            label="Country of residency"
-            placeholder="Start typing location"
-            value={country}
-            onChange={handleCountryChange}
-          />
+          <CreateProfileFormError error={errors.country}>
```

Comment:
**nitpick:** Why `CreateProfileFormError` not `FormInputError`? I think it should be more generic, as it could be used in other parts of the system